res/database/Default/user/kasi.xml

```diff
@@ -176,7 +176,7 @@
 				<job function="1" index="0" required="1" />
 				<job function="16" index="1" gender="2" required="1" />
 			</jobs>
-			<data flags_optional="16" flags="1" production_licence_flags="4" production_broadcast_limit="1" broadcast_time_slot_start="19" broadcast_time_slot_end="23"/>
+			<data flags_optional="16" flags="1" live_date="3,5,19,19" production_licence_flags="4" production_broadcast_limit="1" />
```

Comment:
Damit fixierst Du den Livezeitpunkt auf "Wochentag 5" von "19 - 19 Uhr".
Die Blocklaenge ist "2 - 4".
Urspruenglich war ein Livezeitfenster von 19-23 definiert. Erstellt das Skript ein 2 Stunden-Konzert, waere also eine Ausstrahlung 19, 20 oder 21 Uhr moeglich gewesen. bei 3 Stunden immerhin noch 19 und 20 Uhr.

Reply:
Das ist mir bewusst. Meine Argumentation ist, dass es sich um die Übertragung eines "Fremdlivekonzerts" ist, bei der ich den Live-Startzeitpunkt gar nicht selbst festlegen kann (wie z.B. bei einer eigenen Show). Insofern kann ich mit dieser Variante gut leben.
Wie in#385 beschrieben funktioniert die "Livezeitraum"-Logik auch noch nicht.

Reply:
war also "Absicht". Wollte ich damit nur geklaert haben.